server/post.go

```diff
@@ -48,13 +48,13 @@ func (p *Plugin) buildAnalyticMsg() (string, *model.AppError) {
 
 		m = m + "### :see_no_evil: Podium Channels Conversations\n"
 		if len(data.channels) > 0 {
-			m = m + fmt.Sprintf("* :1st_place_medal: ~%s with a total of **%d** *(%d%%)* messages with %d reply\n", data.channels[0].name, data.channels[0].nb, (data.channels[0].nb*100)/(data.totalMessagesPublic+data.totalMessagesPrivate), data.channels[0].reply)
+			m = m + fmt.Sprintf("* :1st_place_medal: %s with a total of **%d** *(%d%%)* messages with %d reply\n", channelLink(data.channels[0].displayName, data.channels[0].link), data.channels[0].nb, (data.channels[0].nb*100)/(data.totalMessagesPublic+data.totalMessagesPrivate), data.channels[0].reply)
```

Comment:
These lines are way to complicate. Please split this up some way.

Reply:
:heavy_check_mark: